src/nifake/metadata/functions.py

```diff
@@ -1326,6 +1326,14 @@
         'documentation': {
             'description': 'Creates a new IVI instrument driver session.'
         },
+        'method_templates': [
+            {
+                'documentation_filename': 'default_method',
+                'library_interpreter_filename': 'initialization_method',
```

Comment:
It is necessary to specify a different initialization method since it has parameters that are proto-only and not in the functions.py file. I added it manually here so you can see what the code generation looks like. Once the direction is approved, I'll go back to the hapigen metadata, add it there, then re-update this file using an official export.

Reply:
See [my ni-central PR](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/382959) where once submitted, that will allow me to update the metadata here with an official export.

Reply:
Those went in. The latest exported metadata now specifies the new template method for initialize calls.